### ISSUE TITLE
Car Stops streaming while charging

### ISSUE THREAD

**Issue by rawmean** [teslamotors/fleet-telemetry#283]
while charging, the car stops streaming data. Even when the car is awakened, it won’t resume streaming data until a command (eg, turning on climate) is sent to the car. 

This creates a problem because the data in the app becomes out of sync with the car (the car has been charging while streaming stopped).

**Comment by Bre77** [comment]
Your vehicle shouldn't be going to sleep while charging, do you mean while plugged in?

How are you waking your vehicle in your test?

My Model 3 never sleeps while charging and sends Fleet Telemetry as soon as it wakes up.

**Comment by rawmean** [comment]
> Your vehicle shouldn't be going to sleep while charging, do you mean while plugged in?
> 
> How are you waking your vehicle in your test?
> 
> My Model 3 never sleeps while charging and sends Fleet Telemetry as soon as it wakes up.

Exactly, I wasn't expecting the car to go to sleep while charging either. It probably didn't but streaming was stopped for some reason (it was actively charging; not just plugged in).

I woke the car by enabling climate using Tesla app.
